packages/@ama-sdk/schematics/testing/MOCK_package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "fake-package",
   "scripts": {
-    "swagger:regen": "yo ./node_modules/@ama-sdk/generator-sdk/src/generators/java-client-core --swaggerSpecPath ./swagger-spec.yaml --swaggerConfigPath node_modules/@ama-sdk/generator-sdk/src/generators/java-client-core/templates/swagger-codegen-java-client/config/swagger-codegen-config.json --force",
```

Comment:
This file is use for test to apply the changes of ng-add on it 
so you should not modify it I think

Reply:
not if I want to rename the parameter